salt/states/postgres_user.py

```diff
@@ -92,6 +93,9 @@ def present(name,
         encrypted to the previous
         format if it is not already done.
 
+    default_passwoord
+        The password used only when creating the user, unless password is set.
+
```

Comment:
Would you mind adding a `.. versionadded:: Boron` flag here to indicate when this argument was added?